skipper.yaml

```diff
@@ -3,6 +3,8 @@ build-container-image: assisted-installer-build
 
 volumes:
     - $HOME/.cache/go-build:/go/pkg/mod
+    - $HOME/.cache/golangci-lint:$HOME/.cache/golangci-lint
```

Comment:
This should be in the build container.
We want to always use a specific version

Reply:
The cache is a really good feature making our local builds faster. It will always check if there were any changes.
It make the cache really as a cache between different runs, as if you would run it locally.